imports/plugins/included/marketplace/client/components/marketplaceShops.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { Components } from "@reactioncommerce/reaction-components";
+
+class MarketplaceShops extends Component {
+  static propTypes = {
+    onWorkflowChange: PropTypes.func,
+    shops: PropTypes.arrayOf(PropTypes.object)
+  }
+
+  renderShopsTable() {
+    const fields = ["name", "emails", "workflow"];
+
+    const columnMetadata = fields.map((field) => {
+      return {
+        Header: <Components.Translation i18nKey={`marketplaceShops.headers.${field}`} defaultValue={field} />,
+        accessor: field,
+        Cell: (data) => ( // eslint-disable-line
```

Comment:
What's disabled here? It's a little more clear if we can use `eslint-disable-line rule-name` so future us/others can understand why we disabled and what for

Reply:
I'm actually not seeing any lint errors when removing this, so perhaps not needed?

Reply:
It's usually a react issue of defining more than one component per file.